src/app.ts

```diff
@@ -22,10 +23,21 @@ app.get('/', (req, res) => res.render('client'))
 app.get('/login', (req, res) => res.render('login', { error: '' }))
 
 app.post('/login', (req, res) => {
-  if (process.env.ADMIN !== req.body.adminPass)
-    return res.render('login', { error: 'Wrong Admin Password' })
-  res.cookie('adminPass', req.body.adminPass)
-  res.redirect('/admin')
+  const hashPass = (password: string): string => {
```

Comment:
these should be moved into the `auth` folder, created a separate refactoring issue: https://github.com/garageScript/myProxy/issues/120